File: appRvSample/src/main/java/in/til/yogeshkumar/timesanimation/adapter/segment/SampleSegmentView.java
package in.til.yogeshkumar.timesanimation.adapter.segment;

import android.Manifest;
import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.clumob.segment.manager.SegmentViewHolder;

import in.til.yogeshkumar.timesanimation.R;

/**
 * Created by prashant.rathore on 11/07/18.
 */

public class SampleSegmentView extends SegmentViewHolder {

    public SampleSegmentView(Context context, LayoutInflater layoutInflater, ViewGroup parentView) {
        super(context, layoutInflater, parentView);
    }

    @Override
    protected View createView(LayoutInflater layoutInflater, ViewGroup viewGroup) {
        return layoutInflater.inflate(R.layout.item_card_fill, viewGroup, false);
    }

    @Override
    protected void onBind() {
    }

    @Override
    protected void onUnBind() {

    }
}

File: appRvSample/src/main/java/com/timesanimation/adapter/segment/SampleSegmentView.kt
package com.timesanimation.adapter.segment

import com.timesanimation.R
import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.clumob.segment.controller.SegmentController
import com.clumob.segment.manager.SegmentViewHolder

/**
 * Created by prashant.rathore on 11/07/18.
 */
class SampleSegmentView(context: Context, layoutInflater: LayoutInflater, parentView: ViewGroup?)
    : SegmentViewHolder(context, layoutInflater, parentView) {
    override fun createView(layoutInflater: LayoutInflater, viewGroup: ViewGroup?): View {
        return layoutInflater.inflate(R.layout.item_card_fill, viewGroup, false)
    }

    override fun onBind() {}
    override fun onUnBind() {}
}